ICAFIX/hcp_fix_multi_run

```diff
@@ -580,12 +599,9 @@ then
 					# ${hp} needs to be passed in as a string, to handle the hp=pd* case
 					matlab_code="${ML_PATHS} functionhighpassandvariancenormalize(${tr}, '${hp}', '${fmri}', '${Caret7_Command}', '', 1, ${ciftiwisharts}, '${icadimmode}');"
 					log_Msg "Run interpreted MATLAB/Octave with command..."
-					log_Msg "${FSL_FIXDIR}/call_matlab.sh -r ${matlab_code}"
-					${FSL_FIXDIR}/call_matlab.sh -r "${matlab_code}"   # Note: the semicolon within ${matlab_code} needs to be protected with quotation marks
-					;;
-				*)
-					# Unsupported MATLAB run mode
-					log_Err_Abort "Unsupported MATLAB run mode value (FSL_FIX_MATLAB_MODE) in $FSL_FIXDIR/settings.sh: ${FSL_FIX_MATLAB_MODE}"
+					log_Msg "${matlab_interpreter[*]} ${matlab_code}"
+					"${matlab_interpreter[@]}" <<<"${matlab_code}"
```

Comment:
Add the following comment here (taken from `ReApplyFixMultiRun`) regarding the need to use `<<<`?

> 		# Use bash redirection ("here-string") to pass multiple commands into matlab
> 		# (Necessary to protect the semicolons that separate matlab commands, which would otherwise
> 		# get interpreted as separating different bash shell commands)

Reply:
I generally don't copy that comment onto other matlab code, but I can add it to the first interpreted matlab call (above, in the `fmris` loop).